spec/features/notifications/index_page_spec.rb

```diff
@@ -40,7 +40,7 @@ def create_notification(opts = {})
     expect(page).not_to have_content 'You have no notifications yet'
   end
 
-  example 'marking all notifications as read' do
+  example 'marking all notifications as read', :js do
```

Comment:
I didn't know that `js` could be enabled like this in specs.
In other specs I see `js: true`.

Reply:
let's be consistent